src/components/Calculator.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
- [ ] As suggested by the previous reviewer kindly remove this disabling of the linter check. I removed it and it does not return any error - so it is redundant. Kindly remove it.